### ISSUE TITLE
Add support for download events to GA4 and remove support for Google UA

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#3412]
Google's Universal Analytics product has been replaced with GA4. We can now remove support for Google UA from Pressbooks and related plugins.

Also, a holdover from UA that we should address: as reported by eCampusOntario, networks using GA4 are unable to capture download events. They have identified the function to update:

> We have added the GA-4 code but we are unable to capture the downloads event from the pressbooks.
> 
> I did an inspection of the code and found out that we are using the old GA function. Please see the screenshot below.
> 
> ![image](https://github.com/pressbooks/private/assets/44182629/ebb86938-681c-4be8-abc6-e4ebe34cce96)
> 
> Can you please update the function to make it work with the GA-4?
> 
> I have an example how we can do it:
> 
> ![image](https://github.com/pressbooks/private/assets/44182629/77b2ab16-0185-4a59-b240-1b91d8b682e8)
> 
> Please let me know if you have any questions.

(Thomas adds: if there are any questions from the team for the person who wrote in here, I can relay them)

**Comment by richard015ar** [comment]
I updated GoogleAnalytics class to remove multiple versions handling and I left only v4.
Changes in: https://github.com/pressbooks/pressbooks/compare/feat/download-action-ga4?expand=1
To do: update unit tests.

I removed ga3 references in pressbooks network analytics plugin. Changes in: https://github.com/pressbooks/pressbooks-network-analytics/compare/feat/remove-ga3-option?expand=1 

I changed and tested the new way to track download events. Tests were performed in dev.pressbooks.network GA account from my local env. Changes in: https://github.com/pressbooks/pressbooks-book/compare/fix/track-download-ga4?expand=1 

I'll add unit tests and submit PRs as soon as I can.
